src/pages/create/Activity.tsx

```diff
@@ -0,0 +1,374 @@
+import { Button, Stack, TextArea, useDeskproAppClient } from "@deskpro/app-sdk";
+import { useState, useEffect, useReducer } from "react";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useNavigate, useParams } from "react-router-dom";
+import { postData } from "../../api/api";
+import { getObjectsByQuery } from "../../api/api";
+import { ActivitySchema, ActivitySubmit } from "../../types";
+import { DropdownSelect } from "../../components/DropdownSelect/DropdownSelect";
+import {
+  CallSchema,
+  EmailSchema,
+  EventSchema,
+  TaskSchema,
+} from "../../schemas/activity";
+import { useQueryWithClient } from "../../hooks";
+import { QueryKey } from "../../query";
+import { InputWithTitle } from "../../components/InputWithTitle/InputWithTitle";
+import { DateField } from "../../components/DateField/DateField";
+
+const unusableNames = [
+  "Security User",
+  "Chatter Expert",
+  "Data.com Clean",
+  "Platform Integration User",
+  "Automated Process",
+  "Integration User",
+];
+
+const StatusOptions = [
+  {
+    key: "Not Started",
+    value: "Not Started",
+  },
+  {
+    key: "In Progress",
+    value: "In Progress",
+  },
+  {
+    key: "Completed",
+    value: "Completed",
+  },
+  {
+    key: "Deferred",
+    value: "Deferred",
+  },
+  {
+    key: "Waiting on someone else",
+    value: "Waiting on someone else",
+  },
+];
+
+const activityTypes = [
```

Comment:
We need to get this lookup from Salesforce - is there something like a metadata endpoint we can hit to get this? (as above)